### ISSUE TITLE
use tools/train_test_net.py instead of tools/train_net.py

### ISSUE THREAD

**Issue by dyx118186878** [ucas-vg/TinyBenchmark#34]
i test the trained model, it output box.json  but get some error:
![image](https://user-images.githubusercontent.com/43561541/87019349-14ee2d80-c205-11ea-8749-515fcca45c39.png)

**Comment by yinglang** [comment]
Can you provide the detail information of your error?

**Comment by dyx118186878** [comment]
i test the model i trained in single picture，it get correct result .But when i  test it in the dataset, i get the error  
my yaml file is:
MODEL:
  META_ARCHITECTURE: "GeneralizedRCNN"
  WEIGHT: "../params/adap_retina_sm_tinyperson_b4/model_only_model_final.pth"
  RPN_ONLY: True
  RETINANET_ON: True
  BACKBONE:
    CONV_BODY: "R-101-FPN"
  RESNETS:
    BACKBONE_OUT_CHANNELS: 256
  RETINANET:
    NUM_CLASSES: 2
    ANCHOR_SIZES: (4, 8, 16, 32, 64)  # (4, 8, 16, 32, 64) will be better   (8, 16, 32, 64, 128)
    ASPECT_RATIOS: (0.5, 1., 2)
    ANCHOR_STRIDES: (4, 8, 16, 32, 64)
    PRE_NMS_TOP_N: 1000
    BBOX_REG_WEIGHT: 4.0
    LOSS_ALPHA: 0.25 # 0.25
    LOSS_GAMMA: 2.0
    PRIOR_PROB: 0.01
DATASETS:
  TRAIN: ("tiny_set_corner_sw640_sh512_erase_with_uncertain_train_all_coco",)
  TEST: ("tiny_set_corner_sw640_sh512_erase_with_uncertain_train_all_coco",)  #
#  TEST: ("voc_2007_test",)　("tiny_set_corner_sw640_sh512_test_all_coco",)
INPUT:
  USE_SCALE: true
  SCALES: (1,)
DATALOADER:
  SIZE_DIVISIBILITY: 32

# 1 gpu
SOLVER:
  CHECKPOINT_PERIOD: 1603   # 2 epoch
  TEST_ITER: 1603   # 1 epoch
  BASE_LR: 0.005
#  HEAD_LR_FACTOR: 10.
  WEIGHT_DECAY: 0.0001
  STEPS: (9618, 16030)   # (6 epoch, 10 epoch)
  MAX_ITER: 19236 # 12 epoch
  IMS_PER_BATCH: 2
  NUM_GPU: 2
TEST:
  IMS_PER_BATCH: 2
  COCO_EVALUATE_STANDARD: 'tiny'  # tiny need change
#  MERGE_RESULTS: true
#  MERGE_GT_FILE: '/media/ubuntu/0000BFD200076E00/U-net/TinyBenchmark-master/dataset/tiny_set/annotations/corner/task/tiny_set_test_sw640_sh512_all.json'
  IGNORE_UNCERTAIN: true
  USE_IOD_FOR_IGNORE: true
OUTPUT_DIR: ./outputs/tiny_set/retina/baseline2_lrfpn_smb4coco22

i just want to test,  so i change the  TEST same as TRAIN “ tiny_set_corner_sw640_sh512_erase_with_uncertain_train_all_coco“
i can train and get right model  but when test it failed  i donot konw why

**Comment by yinglang** [comment]
OK, thanks, I will try to reproduce this problem. if I have any progress，I will reply here.

**Comment by yinglang** [comment]
I try many times in various computer and settings, I can not reproduce your problem.
And Finally, I read your screenshot of the error again Unintentionally. I found out you seem to use train_net.py to train. I think that may be your problem. please try train_test_net.py to train and test just like section 4. in [here](https://github.com/ucas-vg/TinyBenchmark/tree/master/tiny_benchmark#getting-started-)

**Comment by dyx118186878** [comment]
Thank you very much, that is indeed the reason for the mistake. In addition,, now I am directly using the training set for the test.  if it is convenient for you，Could you or when can you provide the corresponding test set

Thank you again for your kindness

**Comment by yinglang** [comment]
That's fine. For the annotation of test set, we will release when the TOD challenge finished as [this issue](https://github.com/ucas-vg/TinyBenchmark/issues/4) said.
